src/utils/addDelimiter.ts

```diff
@@ -0,0 +1,3 @@
+export default function addDelimiter(str: number) {
+  return str.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
```

Comment:
```suggestion
  return str.toLocaleString();
```
이렇게 쓰는건 어떤가요?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString#Using_toLocaleString

Reply:
엇 바꾸시나요?